style.css

```diff
@@ -90,6 +90,10 @@
     .introduction{
         margin-right: 106px;
     }
+    .intro_img{
+        width: 243px;
+        height: 243px;
+    }
     .intro_head{
```

Comment:
1. `Recent posts` and `View all` shouldn't be that far. Take reference from design

![Screenshot 2023-03-13 at 2 44 44 PM](https://user-images.githubusercontent.com/38748298/224657591-83dd67ff-29fd-435e-9cfa-921e57d25b75.png)

2. The recent post is not having correct margin in mobile view as well